source/reference/minio-mc/mc-ilm-rule-add.rst

```diff
@@ -0,0 +1,420 @@
+.. _minio-mc-ilm-rule-add:
+
+===================
+``mc ilm rule add``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ilm rule add
+
+.. versionchanged:: RELEASE.2022-12-24T15-21-38Z
+
+   ``mc ilm rule rm`` replaces ``mc ilm add``.
+
+
+Description
+-----------
+
+.. start-mc-ilm-rule-add-desc
+
+The :mc:`mc ilm rule add` command adds an object lifecycle management rule to a bucket.
+
+.. end-mc-ilm-rule-add-desc
+
+Required Permissions
```

Comment:
So we have this section in the `mc ilm rule.rst` page, which we could reference. As we should for the other `mc ilm rule` subcommands. That also keeps the Example/Syntax above the fold for each of these pages.

Reply:
I added a new h2 section between the examples and the Behavior section on each page with a link back to the section on the command overview page.
This is a Good Idea :tm:.

Reply:
Also realized Add had a `Description` heading none of the other sub commands had, so I removed it for consistency.